### DISCUSSION TITLE
GBA emulator not keeping saves

### DISCUSSION THREAD

**Discussion by jabadm** [eduardofilo/RG350_adam_image#220] [RetroArch topics]
Default GBA emulator does not keep saves on an RG350. Save states work perfectly but it would be nice to have a working save system to replicate the GBA behavior.

**Comment by eduardofilo** [comment]
In any case it would be an issue of the RetroArch core. Use the alternative core (mGBA) or the standalone emulator (ReGBA). I put gpSP as default because it is the fastest and most convenient for RG350/280 which is a limited machine.
